samples/nrf9160/modem_shell/src/gnss/gnss_interface.c

```diff
@@ -1113,11 +1121,18 @@ int gnss_set_elevation_threshold(uint8_t elevation)
 	err = nrf_modem_gnss_elevation_threshold_set(elevation);
 	if (err) {
 		mosh_error("GNSS: Failed to set elevation threshold");
+	} else {
+		gnss_elevation = elevation;
 	}
 
 	return err;
 }
 
+void gnss_set_agps_filtered_ephemerides(bool enable)
+{
+	gnss_filtered_ephemerides_enabled = enable;
```

Comment:
If `CONFIG_NRF_CLOUD_AGPS_FILTERED_RUNTIME` is not enabled, this should fail and show an error message. See for example `gnss_inject_agps_data()` what happens when A-GPS is not enabled in configuration and "gnss agps inject" command is given.

Maybe we could also enable CONFIG_NRF_CLOUD_AGPS_FILTERED and CONFIG_NRF_CLOUD_AGPS_FILTERED_RUNTIME in the prj.conf by default.